pkg/tsdb/prometheus/models/types_dataquery_gen.go

```diff
@@ -0,0 +1,73 @@
+// Code generated - EDITING IS FUTILE. DO NOT EDIT.
+//
+// Generated by:
+//     public/app/plugins/gen.go
+// Using jennies:
+//     GoTypesJenny
+//     PluginGoTypesJenny
+//
+// Run 'make gen-cue' from repository root to regenerate.
+
+package models
```

Comment:
Here the package name was generated wrong. It was in the `prometheus` folder and should be `prometheus` but it was something else. Also when we run `make run` it was generated again. So even we change it manually it will be overwritten. So to see the result we have to comment out the codegen step in `Makefile`

Reply:
> Here the package name was generated wrong. It was in the prometheus folder and should be prometheus but it was something else.

Yeah, i noticed this when fixing the file location issue. Also fixing that in the same PR.

> Also when we run make run it was generated again. So even we change it manually it will be overwritten.

This is by design. In order for schemas to actually, meaningfully be a single source of truth, they must all be regenerated on every `make gen-cue` execution, and the generated types must be used as-is, without any direct edits to the generated file. (Logical transformations of the generated type are fine, though can clearly get into risky territory. That's what the veneers system are all about.)

This is a strict rule, and if there's a problem with how the code generators work that prevents you from being able to follow it, that's key for us to know so we can fix the issue (as i have with these issues in #61899)! However, we can't say that a kind has advanced to `experimental` maturity until this standard is met, which means it won't show up for use in as-code contexts.